README.md

```diff
@@ -39,13 +41,16 @@ Note: If you intend to build from sources on Linux/macOS, follow [these instruct
 
 Before you build, you will need to have some additional software installed. This is the most straightforward path - see [Building Sources](https://github.com/Microsoft/DirectXShaderCompiler/wiki/Building-Sources) on the Wiki for more options, including Visual Studio 2015 and Ninja support.
 
-* [Git](http://git-scm.com/downloads).
-* [Visual Studio 2017](https://www.visualstudio.com/downloads). Select the following workloads: Universal Windows Platform Development and Desktop Development with C++.
-* [Python](https://www.python.org/downloads/). Version 3.x is required.
+* [Visual Studio 2017](https://www.visualstudio.com/downloads) - select the following workloads: 
+    * Universal Windows Platform Development
+    * Desktop Development with C++
+    * Python development
```

Comment:
I'm assuming this setup includes git? I didn't know that.

Reply:
Yes, git is included in Visual Studio.

Reply:
I'm putting the Git and Python requirement back in. I have checked and hctstart.cmd does not find the git or python in Visual Studio. So I have uninstalled the standalone git from my machine and updated hctstart.cmd to get the git from VS.
Then I have tried working with it a bit but I keep running into little glitches and differences and then it did not let me push the changes up. So I am backup up. The standalone Git install will be more reliable and easier to update. Same for python.

Reply:
Kind of a shame. Would have been nice to keep the instructions that simple. Maybe we'll figure out how to do it later on.